modules/installation-custom-aws-vpc.adoc

```diff
@@ -36,14 +39,18 @@ If you use a cluster with public access, you must create a public and a private
 
 The installation program modifies your subnets to add the `kubernetes.io/cluster/.*: shared` tag, so your subnets must have at least one free tag slot available for it. Review the current link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#tag-restrictions[Tag Restrictions] in the AWS documentation to ensure that the installation program can add a tag to each subnet that you specify.
 
-If you are working in a disconnected environment, you are unable to reach the
-public IP addresses for EC2 and ELB endpoints. To resolve this, you must create
-a VPC endpoint and attach it to the subnet that the clusters are using. The
-endpoints should be named as follows:
-
+ifndef::aws-china[]
+If you are working in a disconnected environment, you are unable to reach the public IP addresses for EC2 and ELB endpoints. To resolve this, you must create a VPC endpoint and attach it to the subnet that the clusters are using. The endpoints should be named as follows:
```

Comment:
refer to ELB endpoint comment https://github.com/openshift/openshift-docs/pull/35550#issuecomment-919659742

Reply:
@yunjiang29 This is an excellent catch. I could have sworn that I had read that ELB is not supported, but their doc clearly states that it is.

Reply:
@mjpytlak No.
There are no `.cn` in S3 and ELB endpoint.
* EC2: `ec2.<region>.amazonaws.com.cn`
* S3: `s3.<region>.amazonaws.com`
* ELB: `elasticloadbalancing.<region>.amazonaws.com`